graph/src/data/subgraph/schema.rs

```diff
@@ -214,6 +214,8 @@ pub struct SubgraphDeploymentEntity {
     ethereum_head_block_hash: Option<H256>,
     ethereum_head_block_number: Option<u64>,
     total_ethereum_blocks_count: u64,
+    entity_count: u64,
```

Comment:
Let's keep these out of of the main deployment entity. How about we introduce a separate entity, like with the deployment assignments: `SubgraphDeploymentPricing`?

Reply:
Actually, `entityCount` does belong here, but the price doesn't. The only question I have: does anything actually set the entity count?